src/odo.ts

```diff
@@ -361,14 +373,25 @@ export class OdoImpl implements Odo {
     public async _getComponents(application: OpenShiftObject): Promise<OpenShiftObject[]> {
         const result: cliInstance.CliExitData = await this.execute(Command.listComponents(application.getParent().getName(), application.getName()));
         let data: any[] = [];
+        let compSource = '';
```

Comment:
Should be  declared inside array.map callback on line 384.`

Reply:
Fixed.